tuhi-gui.in

```diff
@@ -17,16 +17,25 @@ from gi.repository import Gio
 
 
 @devel@  # NOQA
-resource = Gio.resource_load(os.fspath(Path('@pkgdatadir@', 'tuhi.gresource')))
+if "TUHI_DEVEL" in os.environ:
+    resource_path = Path(os.environ["TUHI_DEVEL"], 'data', 'tuhi.gresource')
+else:
+    resource_path = Path('@pkgdatadir@', 'tuhi.gresource')
+    
+ resource = resource = Gio.resource_load(os.fspath(resource_path))
```

Comment:
```suggestion
resource  = Gio.resource_load(os.fspath(resource_path))
```

leading space that snuck in somehow, and an unnecessary duplicate assignment. guess that's what I get for typing diffs into github directly :)

Reply:
fixed